backend/Squirrel.SqlService/Squirrel.SqlService.WebApi/appsettings.json

```diff
@@ -17,5 +17,6 @@
   "AllowedHosts": "*",
   "UserDbChangesBlobContainerName": "user-db-changes",
   // Add path to python.exe  Used for registrating SqlFormatterService
-  "PythonExePath": "C:\\Python310\\python.exe"
+  "PythonExePath": "C:\\Python310\\python.exe",
+  "SqlServiceRoute": "https://localhost:7244"
```

Comment:
Why you need SQL Service in SQL Service?

Reply:
To make calls to controller, is this wrong approach?

Reply:
you call this one from 'Core` web api